README.md

```diff
@@ -1,5 +1,15 @@
 # Smart Contract Backend
 
+Run off-chain smart contract executables server-side in isolation, accessible via a GraphQL interface. The [server](src/server/README.md) exposes a GraphQL control API for loading contracts and subscribing to signing requests for transactions generated by the contracts. An [execution service](src/execution_service/README.md) isolates potentially untrusted code execution, enabling interaction the contracts via a [Docker](src/execution_service/infrastructure/execution_engines/DockerExecutionEngine.ts) or [NodeJS](src/execution_service/infrastructure/execution_engines/NodeJsExecutionEngine.ts) engine.
```

Comment:
"enabling interaction the contracts" -> "enabling interaction with the contracts"